README.md

```diff
@@ -47,24 +71,18 @@ client_secret="your_client_secret" #Replace this value by the client secret of y
 
 sdk = SDK(client_id=client_id, client_secret=client_secret)
 
-# Creation of a sequence
-reg = Register.square(2, prefix="q")
-sequence = Sequence(reg, devices.Chadoq2)
-# You should add channels and variables to your sequence
-# See Pulser documentation for details
-serialized_sequence = sequence.serialize() # Serialize the sequence in json format
+# Set the wanted values for the variables defined in the sequence
+job1 = {"runs": 20, "variables": {"omega_max": 6} }
+job2 = {"runs": 50, "variables": {"omega_max": 10.5} }
 
-# Create a new batch and send it to Pasqal QPUs
-batch = sdk.create_batch(serialized_sequence)
-# You can also choose to run your batch on an emulator using the optional argument 'emulator'
-# batch = sdk.create_batch(serialized_sequence, emulator=True)
+# Send the batch of jobs to the QPU and wait for the results
+batch = s.create_batch(serialized_sequence, [job1,job2], wait=True)
```

Comment:
nit `sdk` ? We should probably test that exact script by the way  :) Should we add it to the end-to-end tests ?

Reply:
(not nit sorry :D )

Reply:
Yes done.. I tested it initially and forgot to fix that issue my bad.. I'll rerun a test

Reply:
We already have a test very similar to this in end to end tests so not sure it brings much value to add it explicitly there. 

What we could do maybe is to create an `examples` folder (which could be used as tutorial for our users) in this repo that we would run in the e2e-tests. Thoughts ?